52n-xml-sosdo-v20/src/main/xsd/sosdo/2.0/sosdo.xsd

```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    OGC LICENSE HEADER
+
+    The official OGC XML schemas can be accessed under http://schemas.opengis.net.
+    Please consider the schema copyright notices: http://www.opengeospatial.org/ogc/document
+    and the OGC Copyright Notice and Disclaimers: http://www.opengeospatial.org/ogc/legal
+    If there are any questions left there is also a copyright FAQ: http://www.opengeospatial.org/ogc/legalfaq
+
+-->
+<schema xmlns="http://www.w3.org/2001/XMLSchema" xmlns:gml="http://www.opengis.net/gml/3.2" xmlns:fes="http://www.opengis.net/fes/2.0" xmlns:sosdo="http://www.opengis.net/sosdo/2.0" xmlns:swes="http://www.opengis.net/swes/2.0" targetNamespace="http://www.opengis.net/sosdo/2.0" elementFormDefault="qualified" attributeFormDefault="unqualified">
+	<import namespace="http://www.opengis.net/swes/2.0" schemaLocation="http://schemas.opengis.net/swes/2.0/swes.xsd"/>
+	<import namespace="http://www.opengis.net/gml/3.2" schemaLocation="http://schemas.opengis.net/gml/3.2.1/gml.xsd"/>
+    <import namespace="http://www.opengis.net/fes/2.0" schemaLocation="http://schemas.opengis.net/filter/2.0/filterAll.xsd"/>
+	<!-- ================================================= -->
+    <element name="DeleteObservation" type="sosdo:DeleteObservationType" />
+    <complexType name="DeleteObservationType">
+        <complexContent>
+            <extension base="swes:ExtensibleRequestType">
+                <sequence>
+                    <choice>
+                        <element name="observation" type="anyURI" minOccurs="0" maxOccurs="unbounded">
+                            <annotation>
+                                <appinfo>
+                                    <gml:targetElement>unknown</gml:targetElement>
+                                </appinfo>
+                                <documentation>Pointer to the observation that shall be deleted.</documentation>
+                            </annotation>
+                        </element>
+                        <sequence>
+                            <element name="procedure" type="anyURI" minOccurs="0" maxOccurs="unbounded">
+                                <annotation>
+                                    <documentation>pointer to a procedure for which time series shall be discovered
```

Comment:
It is a copy&paste issue in the comment and should be `deleted`.
I will fix it as soon as possible.